### PR TITLE
Editor optimizations

### PR DESCRIPTION
**Pull request by Therzok**
Take the second commit with a grain of salt, as I suck at naming. Any suggestion on what to replace 'SomethingSomething' with?

cc @DavidKarlas this fixes the issue you've been having.
cc @slluis and @mkrueger for sanity reviews.

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/SdiWorkspaceWindow.cs [hunk 1]
```diff
@@ -110,8 +108,6 @@ public SdiWorkspaceWindow (DefaultWorkbench workbench, ViewContent content, Dock
 
 			// The previous WorkbenchWindow property assignement may end with a call to AttachViewContent,
 			// which will add the content control to the subview notebook. In that case, we don't need to add it to box
-			if (subViewNotebook == null)
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/SdiWorkspaceWindow.cs]: I'm not sure if this can/will break anything.

**Comment by DavidKarlas** [comment]
Maybe "SomethingSomething" could be replaced with "realize", "realized"... Similar to what GTK has...

**Comment by Therzok** [comment]
Renamed to use Realized instead of SomethingSomething.

**Comment by slluis** [comment]
LGTM

**Pull request merged**